dev/com.ibm.ws.repository.resolver/src/com/ibm/ws/repository/resolver/RepositoryResolver.java

```diff
@@ -202,7 +234,7 @@ public RepositoryResolver(Collection<ProductDefinition> installDefinition,
      * <p>This will also return a list of resources for any auto features that have their required provision capabilities satisfied either by what is already installed, the new
      * features being installed or a combination of the two. Note that this means auto features that are in no way related to what you have asked to resolve could be returned by
      * this method if they are satisfied by the current installation.</p>
-     * 
+     *
```

Comment:
I've done that, and we also noted that for `resolveAsSet` you need to provide the full set of features from the `server.xml`, including any installed features, otherwise you're not guaranteed to get a correct result.